### ISSUE TITLE
Plans: cannot scroll plan details table when using VoiceOver

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12856]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to Plans and tap on any plan to go to the details screen.
1. Tap on an item in the table.
1. Try to scroll through the table by swiping up with three fingers.

**Result:** the table won't scroll. ([21s](https://cloudup.com/cHxHp51-5Nh))

<img src="https://user-images.githubusercontent.com/1119271/68171570-861c6900-ff31-11e9-9909-2b979ffb158f.png" width="270" alt="cannot-scroll-plan-details-table-when-using-voiceover">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
